File: Tests/TestData/MultiFileCharacterization/SourceFiles/WindowsAppVb/ReferencingFormThroughStatic.vb
Public Module ReferencingFormThroughStatic
    Public Function GetFormTitle() As String
        ' This used to cause a bug in the expander for maes.Expression leading to another bug later on
        With New System.Text.StringBuilder()
            .Capacity = 4
        End With

        If WinformsDesignerTest IsNot Nothing AndAlso WinformsDesignerTest.Text IsNot Nothing Then
            Return WinformsDesignerTest.Text
        End If
        Return ""
    End Function
End Module

File: Tests/TestData/MultiFileCharacterization/VBToCSResults/ConvertWholeSolution/WindowsAppVb/ReferencingFormThroughStatic.cs

namespace WindowsAppVb
{
    public static class ReferencingFormThroughStatic
    {
        public static string GetFormTitle()
        {
            // This used to cause a bug in the expander for maes.Expression leading to another bug later on
            {
                var withBlock = new System.Text.StringBuilder();
                withBlock.Capacity = 4;
            }

            if (My.MyProject.Forms.m_WinformsDesignerTest is object && My.MyProject.Forms.WinformsDesignerTest.Text is object)
            {
                return My.MyProject.Forms.WinformsDesignerTest.Text;
            }

            return "";
        }
    }
}